src/SLCore/Core/IRpcDebugger.cs

```diff
@@ -36,31 +36,29 @@ public interface IRpcDebugger
 internal class RpcDebugger : IRpcDebugger
 {
     private readonly IFileSystem fileSystem;
-    public string LogFilePath { get; set; } = null;
+    private readonly string logFilePath;
 
     [ImportingConstructor]
-    public RpcDebugger() : this(new FileSystem())
+    public RpcDebugger() : this(new FileSystem(), DateTime.Now)
```

Comment:
Why?

Reply:
Timezones can change. However unlikely it is, it's better to use UTC since it's stable

Reply:
We don't care about timezones, we do not care about exact time this happened.

Reply:
Actually because of timezones it could be confusing people would get logs with different date than they are in.

Reply:
You can also add UTC at the end

Reply:
It would be very seldom that a developer moves timezones and develop in the same day that would affect the order of the files. Only scenario I can think is if somebody crosses international date line and have change time zone automatically set up. 
Thinking having different dates with UTC would be around midnight is a very eurocentric approach, it's probable to have this scenario for a developer in California, Japan or New Zealand.